baseplate/events/queue.py

```diff
@@ -21,8 +21,25 @@
 import time
 import uuid
 
+from enum import Enum
+
 from . import MAX_EVENT_SIZE, MAX_QUEUE_SIZE
 from baseplate.message_queue import MessageQueue, TimedOutError
+from baseplate._utils import warn_deprecated
+
+
+class FieldType(Enum):
```

Comment:
since this goes into a param called `kind`, maybe `FieldKind` would be more consistent?